### ISSUE TITLE
[Bug]: Tray Icon not Working on Fedora 39 & GNOME

### ISSUE THREAD

**Issue by alexknop** [nextcloud/desktop#6206]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Fedora 39 (released 11/7) comes with a newer version of GNOME (45.1) and now the tray icon for Nextcloud is not working.
First, the tray icon will not show by default. I don't see any officially documented solutions for this but after some research I installed the GNOME extension "Tray Icons: Reloaded" and now I am seeing the icon,
![image](https://github.com/nextcloud/desktop/assets/11878115/31b20118-fa60-4c86-a44a-241b39797c1e)

However when I right click on it, the options are showing on the left side of the screen:
![image](https://github.com/nextcloud/desktop/assets/11878115/b0f18ebd-2d79-4070-bcaa-2014056097d1)

It doesn't appear there is an official method of implementing a tray icon for GNOME at the moment, so Nextcloud documentation should be updated to reflect updates to GNOME and workarounds or the status of the tray icon needs to be laid out to Linux users. As of now, the documentation makes it seem users should expect a visual tray icon that has options that will appear underneath it if you right-click.

### Steps to reproduce

1. Install Fedora 39 Workstation
2. Install Nextcloud Desktop Client
3. Tray icon will not show
4. (Optional) Use Tray Icons: Reloaded GNOME extension to force tray icon to appear. Right-clicking on the tray icon won't have its options show directly underneath.

### Expected behavior

On previous Fedora/GNOME versions, the tray icon options would load directly underneath the icon.

### Which files are affected by this bug

NA

### Operating system

Linux

### Which version of the operating system you are running.

Fedora 39

### Package

Appimage

### Nextcloud Server version

27.1.2

### Nextcloud Desktop Client version

3.10.1 (Fedora)

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by Freeedim** [comment]
I don't know whether I should open a new issue or just comment here, but I have recently started having the same issue (no icon in the system tray) with KDE Plasma on a Debian Testing.
Plasma 5.27.9
KDE Frameworks 5.107.0
Qt 5.15.10
Graphics platform: X11
Kernel 6.5.0

If I want to open the application, I need to launch it from a terminal by typing `nextcloud` or double click on the .desktop file.

**Comment by a73s** [comment]
works for me with native client fedora 39 gnome

**Comment by alexknop** [comment]
@a73s which GNOME extension are you using to get the tray icon to show?

**Comment by a73s** [comment]
> @a73s which GNOME extension are you using to get the tray icon to show?

"AppIndicator and KStatusNotifierItem Support"

**Comment by janvlug** [comment]
I also use https://extensions.gnome.org/extension/615/appindicator-support/, and it works fine for me on Fedora 39, GNOME Wayland desktop.

**Comment by alexknop** [comment]
Yeah I didn't realize I had to reboot but the AppIndicator and KStatusNotifierItem Support extension made it work for me. The other extension must be broken.
However, still requesting Nextcloud provide some sort of documentation on this. Linux users shouldn't be left in the dark, even if we're used to having to research and put the puzzle pieces together ourselves.

**Comment by dinhphu28** [comment]
> I also use https://extensions.gnome.org/extension/615/appindicator-support/, and it works fine for me on Fedora 39, GNOME Wayland desktop.

Yeah, It work  for me, thanks!

**Comment by ASerbinski** [comment]
I think that the issue is that the newer versions of gnome don't actually have a traditional "system tray" or anything that even remotely works like one. While some things do add pieces to the header bar as if they were systray icons, those things actually include their own gnome-shell extensions.

**Comment by claucambra** [comment]
Since this is a GNOME issue I will close... Thanks for the reports!
